packages/in_app_purchase/in_app_purchase_android/lib/src/billing_client_wrappers/sku_details_wrapper.dart

```diff
@@ -19,6 +20,13 @@ part 'sku_details_wrapper.g.dart';
 const String kInvalidBillingResultErrorMessage =
     'Invalid billing result map from method channel.';
 
+/// Abstraction of result of [BillingClient] operation that includes
```

Comment:
Is this class in the correct file? I expected this to live in the billing_client_manager.dart file.